sympy/polys/densetools.py

```diff
@@ -83,24 +112,33 @@ def dmp_integrate(f, m, u, K):
     >>> R.dmp_integrate(x + 2*y, 2)
     1/6*x**3 + x**2*y
 
+    >>> from sympy.polys.densetools import dmp_integrate
+    >>> # Integration of x+2*y+z by x, Result: 1/2*x**2 + 2*x*y + x*z
+    >>> dmp_integrate([[[1]], [[2], [1, 0]]], 1, 2, QQ)
+    [[[1/2]], [[2], [1, 0]], [[]]]
+
+
     """
     if not u:
         return dup_integrate(f, m, K)
 
     if m <= 0 or dmp_zero_p(f, u):
         return f
 
-    g, v = dmp_zeros(m, u - 1, K), u - 1
+    result = dmp_zeros(m, u - 1, K)
+    reduced_vars = u-1
 
-    for i, c in enumerate(reversed(f)):
-        n = i + 1
+    for i, coefficient in enumerate(reversed(f)):
+        # The denominator represent also the exponent
+        denominator = i + 1
```

Comment:
Please don't rename all the variables like this. It is churn and doesn't actually make the code any easier to understand.